### PR TITLE
Handling exception when judgment not found.

### PR DESCRIPTION
**Pull request by pawelczak**
#583

### CODE REVIEW

**Comment by lukdumi** [comment]
No, no, no. It's almost all wrong. You SHOULDN'T create yet another service for getting judgment, it's a mess. And you shouldn't throw ObjectNotFoundException which is hibernate dependend but jpa EntityNotFoundException (as I wrote you yesterday).

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/DefaultExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+@ControllerAdvice
+public class DefaultExceptionHandler {
+	
+	
+	@ExceptionHandler(EntityNotFoundException.class)
+	public String entityNotFoundExceptionErrorHandler(EntityNotFoundException e) {
+		e.printStackTrace();
+		return "judgmentNotFound";
```
Comment (lukdumi) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/DefaultExceptionHandler.java]: It should be more general I think, e.g. entityNotFound. What if the exception will be thrown during getting other object types from db?

File: saos-webapp/src/main/webapp/WEB-INF/view/error/judgmentNotFound.jsp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<%@page pageEncoding="UTF-8"%>
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/view/error/judgmentNotFound.jsp]: The page should be more generic, e.g. entityNotFound

File: saos-webapp/src/main/resources/message/common_pl.properties [hunk 3]
```diff
@@ -6,6 +6,7 @@ page.lang = pl
 
 error.pageNotFound = Brak strony pod danym adresem
 error.internal = Wystąpił błąd aplikacji
+error.judgmentNotFound = Brak orzeczenia o podanym identyfikatorze
```
Comment (lukdumi) [inline saos-webapp/src/main/resources/message/common_pl.properties]: error.entityNotFound = Brak obiektu o podanym identyfikatorze

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/DefaultExceptionHandler.java (hunk 1 above)
Comment (pawelczak) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/DefaultExceptionHandler.java]: :+1:

File: saos-webapp/src/main/webapp/WEB-INF/view/error/judgmentNotFound.jsp (hunk 2 above)
Comment (pawelczak) [inline saos-webapp/src/main/webapp/WEB-INF/view/error/judgmentNotFound.jsp]: :+1:

File: saos-webapp/src/main/resources/message/common_pl.properties (hunk 3 above)
Comment (pawelczak) [inline saos-webapp/src/main/resources/message/common_pl.properties]: :+1:

**Pull request closed without merging**